scanpy/plotting/_docs.py

```diff
@@ -31,7 +31,8 @@
 edges_color : matplotlib color(s), optional (default: `'grey'`)
     Color of edges. See :func:`~networkx.drawing.nx_pylab.draw_networkx_edges`.
 arrows
-    Show arrows (requires to run :func:`~scanpy.tl.rna_velocity` before).
+    Show arrows (requires to run :func:`scvelo.tl.velocity_embedding` before).
```

Comment:
@VolkerBergen is `scvelo.tl.velocity_embedding` correct here?

Reply:
Perfectly done. Keeping the plotting functionality while pointing towards the new outsourced modules. 

`scvelo.tl.velocity_embedding` is correct while requiring some preceding steps (such as `tl.velocity` and `tl.velocity_graph`). However, since it is raising a message of required preceding steps, `tl.velocity_embedding` might be the right one to put in here. Alternatively, simply scvelo (instead of specifying velocity_embedding) would probably suffice as well.

Reply:
OK, great!